tts-evaluation-MCD-DTW.ipynb

```diff
@@ -0,0 +1,433 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "x_axis = np.linspace(1, len(mcds_m1), 10)\n",
+    "plt.plot(x_axis, mcds_m1, label=\"model1\")\n",
+    "plt.plot(x_axis, mcds_m2, label=\"model2\")\n",
+    "\n",
+    "plt.title(\"MCD DTW value for each file\")\n",
+    "plt.ylabel(\"MCD_DTW value\")\n",
+    "\n",
+    "plt.grid()\n",
+    "plt.legend()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ab300bf",
+   "metadata": {},
+   "source": [
+    "## Conclusion\n",
+    "\n",
+    "From the graph above and the average MCD across files. We can conclude that model1 has better convergence than model2. MCD is a great tool to analyse the model convergence. But, the final quality of model cannot be trusted based on MCD as it tends to pleateaus with more training and difference between two MCD values is not usually very high."
```

Comment:
`From the graph above and the average MCD across files. We can conclude` Can you go into more depth about how you made this conclusion?